plone/app/linkintegrity/browser/info.py

```diff
@@ -55,7 +55,10 @@ def get_breaches(self, items=None):
             # list of uids that are ignored
             uids_to_ignore.extend([i.UID for i in brains_to_delete])
             for brain_to_delete in brains_to_delete:
-                obj_to_delete = brain_to_delete.getObject()
+                try:
+                    obj_to_delete = brain_to_delete.getObject()
+                except (AttributeError, KeyError):
+                    continue
```

Comment:
In my case it was a Solr which is out of sync but could be any catalog entry. Logging could be a bit tricky because the claims to have an record which it actually has not and no access to its data.